ta/lessons/basics/collections.md

```diff
@@ -1,6 +1,6 @@
-﻿---
+---
```

Comment:
Thanks for pointing this out, @kalarani! The problem was caused by BOM, so I've removed it from the lessons containing it in #1241.